### DISCUSSION TITLE
Zoom Webinar / Class

### DISCUSSION THREAD

**Discussion by Ampurcell** [cortexrd/Knack-Toolkit-Library#204] [Ideas]
Hello Norm, 

Ever thought about having a how to Webinar or something. The video tutorials are great and I don't think I would have ever figured out how to even start without them but this is still not my strong suit. I feel like I lack some basic understanding of the coding. Maybe their are some others like me. I'd pay!

**Comment by CSWinnall** [comment]
Hi what areas do you think you'd like to see covered?

**Reply by Ampurcell** [reply]
Honestly a review of the wiki and setting up the dev mode.

**Reply by CSWinnall** [reply]
We would be more than happy to help you get the dev mode set up if you'd like.  If you don't mind coding, it's a game changer.  It saves us hours of extra work. Let us know if you're interested and we can set up a meeting.

Craig & Amanda

**Reply by Ampurcell** [reply]
Yes, I would definitely be interested.

**Reply by CSWinnall** [reply]
Hi 
Just out of interest do you use a Mac or PC? We have set up dev on both.
What time zone are you in?

Craig
